pos_order_to_sale_order/static/src/xml/pos_order_to_sale_order.xml

```diff
@@ -1,12 +1,26 @@
 <?xml version="1.0" encoding="utf-8"?>
 <template>
 
-    <t t-name="CreateSaleOrderButtonWidget">
-        <br />
-        <div style="text-align: center">
-            <button class="input-button">
-                <t t-esc="widget.display_text"/>
-            </button>
+    <t t-name="DeliveryButtonWidget">
+        <t t-set="isPicking" t-value="widget.isPicking()" />
+        <div t-attf-class='button #{ isPicking ? "highlight": ""}'>
+            <i t-attf-class='fa #{ isPicking ? "fa-arrow-circle-down": "fa-forward"}' />
+            Delivery 
+            <t t-if="!isPicking"> Later</t>
+            <t t-if="isPicking"> Now</t>
```

Comment:
I think this is uneeded. Just a "Deliver now" text should be shown. If the button is active or not is going to give de clue of the expected behaviour.